sources/academy/platform/deploying_your_code/inputs_outputs.md

```diff
@@ -75,7 +75,7 @@ await Actor.init();
 
 const { numbers } = await Actor.getInput();
 
-const addAllNumbers = (...nums) => nums.reduce((total, curr) => (total += curr));
+const addAllNumbers = (...nums) => nums.reduce((total, curr) => (total + curr), 0);
```

Comment:
not that it would matter much, but imho the braces around function body make things less readable in the end

funny that we had missing params like this in the examples, great that we can now polish this

Reply:
Went too fast to even spot this :) removed the parens now